src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogComputerPublisher.java

```diff
@@ -80,33 +79,43 @@ protected void doRun() throws Exception {
             // Add JenkinsUrl Tag
             globalTags = TagsUtil.addTagToTags(globalTags, "jenkins_url", DatadogUtilities.getJenkinsUrl());
             for (Computer computer : computers) {
+                Map<String, Set<String>> tags = TagsUtil.merge(
+                        DatadogUtilities.getComputerTags(computer), globalTags);
                 nodeCount++;
+                boolean online = true;
                 if (computer.isOffline()) {
+                    online = false;
```

Comment:
Would it make sense to submit the metric here?
```suggestion
                   client.gauge("jenkins.node_status.offline", isOffline, hostname, tags);
```

Reply:
Hm that is true- both the offline and online metrics aren't needed. These metrics were based off of the `jenkins.node.*` metrics though, and there is both a ` jenkins.node.offline`  and a  `jenkins.node.online`, so it would be inconsistent if we submitted only one of them for `node_status`. It's definitely worth considering though.

Reply:
`jenkins.node.offline` and `jenkins.node.online` are total counts vs standalone status. It does work to immediately query a metric for a node to see if its status. I was thinking in terms of metrics like a service check, we only use `can_connect` to determine 🟢  or 🔴

Reply:
That's true! It could even be named something different like `jenkins.node_status.up` or something like that
cc @FlorianVeaux

Reply:
I think this is a great idea, this will work exactly the same way as the `datadog.agent.up` metric.
So 👍

Reply:
Updated! Thanks for the suggestion @ChristineTChen !